pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -294,6 +294,13 @@ func translateRoute(in *networking.HTTPRoute,
 		return nil
 	}
 
+	path := getMatchPath(match)
```

Comment:
Suggestion: define a function on envoy route that returns a string. We can consolidate all the heuristics there to provide a human-readable description of the route.

Reply:
Done.